Firmware/Marlin_main.cpp

```diff
@@ -7957,6 +7968,14 @@ SERIAL_PROTOCOLPGM("\n\n");
                     else if(code_seen('Q'))
                          SERIAL_PROTOCOLLN(GCODE_LEVEL);
                     break;
+               case ClPrintChecking::_Features:  // ~ .6
+                    break;
+               case ClPrintChecking::_PrinterState: // ~.7
+                    if(code_seen('P'))
+                         printer_state=code_value_uint8();
+                    else if(code_seen('Q'))
+                         SERIAL_PROTOCOLLN((int)printer_state);
+                    break;
```

Comment:
I think this should be in https://github.com/prusa3d/Prusa-Firmware/pull/4429 it's missing the `~.7`?